lib/server-builder.js

```diff
@@ -163,6 +163,7 @@ class ServerBuilder {
     const gradlebuild = new SubProcess(cmd, args, {
       cwd: this.serverPath,
       stdio: ['ignore', 'pipe', 'pipe'],
+      windowsVerbatimArguments: true
```

Comment:
May help...?
https://nodejs.org/api/child_process.html#child_processspawncommand-args-options

Reply:
https://github.com/appium/appium-espresso-driver/issues/752#issuecomment-1053029534 
Ok, so, with this one, at least `'` in Windows env will gone. Then, they can provide `"xxxx"` as the capability to avoid 
 the Windows's escape error....

Reply:
So, with this PR and `"START&END"` as the capability, the 
```
END is not recognized as an internal or external command,
```
error should fix.

Reply:
I'd like to add `"` automatically on Windows case after this merge and ask the reporter to try this